examples/blog-starter/package.json

```diff
@@ -10,6 +10,7 @@
     "date-fns": "^3.3.1",
     "gray-matter": "^4.0.3",
     "next": "14.1.0",
+    "nextjs-darkmode": "^1.0.1",
```

Comment:
Can we avoid adding dependencies for this?

Reply:
This dependency brings the following features

- Sync theme across browsing context - tabs or iframes
- Persist theme in localStorage
- Apply theme to all pages without FOUC
- Provide a handy toggle button for switching themes
- Provides `useMode` hook (which is available in all client components without any wrapper/context provider) for full control

I think it's good idea to extract the engineering, e.g, injecting scripts, global store, etc. in a separate library.

Reply:
I hear you, however, there is bias here since you also created that dependency. Let's keep this example as simple as possible and if you would like to make a forked version which includes your library, that would be amazing on your GitHub. Thank you 🙏

Reply:
Ok sir.